packages/website/ts/components/onboarding/portal_onboarding_flow.tsx

```diff
@@ -89,8 +90,12 @@ export class PortalOnboardingFlow extends React.Component<PortalOnboardingFlowPr
     }
 
     private _userHasWeth(): boolean {
-        // TODO: https://app.asana.com/0/681385331277907/690722374136933
-        return false;
+        const ethToken = utils.getEthToken(this.props.tokenByAddress);
+        if (!ethToken) {
+            return false;
+        }
+        const wethTokenState = this.props.trackedTokenStateByAddress[ethToken.address];
```

Comment:
the `TokenState` has a `isLoaded` property that may be false, do we want to handle that case? The user may have WETH but the balance is not updated yet

Reply:
I just renamed the fn to be _hasVisibleWeth